entity-framework/core/querying/complex-query-operators.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Complex Query Operators - EF Core
+author: smitpatel
+ms.date: 09/30/2019
+ms.assetid: 2e187a2a-4072-4198-9040-aaad68e424fd
+uid: core/querying/complex-query-operators
+---
+# Complex Query Operators
+
+Language Integrated Query (LINQ) contains many complex operators which combines multiple data sources or does complex processing. Not all LINQ operators have suitable translation in server side. Sometimes, query written in one form can be translated but in different form doing different computation cannot be translated. This page describes some of the complex operators and their supported variations. In future we may recognized more pattern and add their translation. Important point to remember is database also varies in what they support so a particular query which is translated in SqlServer may not work for Sqlite database.
```

Comment:
```suggestion
Language Integrated Query (LINQ) contains many complex operators which combines multiple data sources or does complex processing. Not all LINQ operators have suitable translations on the server side. Sometimes, query written in one form can be translated but in different form it can't be even if the result is same. This page describes some of the complex operators and their supported variations. In future we may recognize more patterns and add their translations. Important point to remember is that database capabilities also vary, so a particular query which is translated on SqlServer may not work for a Sqlite database.
```